packages/tx/src/constants.ts

```diff
@@ -0,0 +1,11 @@
+/** EIP4844 constants */
+
+// TODO: Decide which of these should be in the hardfork params instead of hardcoded as constants here
```

Comment:
I would cautiously think that we might be ok here for now (constants here  vs. in Common), all these upper bounds constants likely won't change and the other two might also not be such an imminent target for change?

If you agree I think the TODO can be removed.

I also won't cost us much and has no big compatibility issues if we at some point add a constant as a parameter to Common later on and then add a deprecation note here.

Reply:
once the withdrawals PR it merged, it will introduce units module as suggested by @holgerd77 , we can move the constants there

Reply:
Okay, will leave them here for now.

Reply:
This is outdated, misunderstanding between Gajinder and me (units should stay in Util).